.gitignore

```diff
@@ -9,3 +9,4 @@ bazel-out
 bazel-testlogs
 bazel-driver
 gidari-cli
+.idea
```

Comment:
@cecobask Can you remove this from the `.gitignore` file?

Reply:
Done 👍🏻
I saw there are VSCode IDE files being ignored and thought it'd help prevent committing IDE files from [GoLand](https://www.jetbrains.com/go/) as well.

Reply:
Ah I see, I didn't realize what that was. I should have asked, I will create a follow up ticket to extend the .goignore file. Thank you.

Reply:
No problem 🙂 Is there anything else I can do for #318?